src/basic-languages/python/python.ts

```diff
@@ -271,10 +287,24 @@ export const language = <languages.IMonarchLanguage>{
 			[/'/, 'string.escape', '@popall'],
 			[/\\$/, 'string']
 		],
-		fDblStringBody: [
+		fDblStringBody3: [
+			[/[^\\"\{\}]+/, 'string'],
+			[/\{\{[^}]+\}\}/, 'string'],
+			[/\{f'{3}/, 'string', '@fStringBody3'], // for nested f-strings
+			[/\{f'{1}/, 'string', '@fStringBody1'], // for nested f-strings
+			[/\{[^\}'":!=]+/, 'identifier', '@fStringDetail'],
+			[/\\./, 'string'],
+			[/"""/, 'string.escape', '@popall'],
+			[/"/, 'string.escape'],
```

Comment:
So we escape on the `"` and on the `"""`. What use case does it fix? And why does line 298 not have `@popall` as third value in the array?